### PR TITLE
Improving the electron wrapper 

### PR DESCRIPTION
**Pull request by amitojsingh366**
- Fixed issue where environmental variables weren't working in the built app
- Fixed issue where the app would mess with the OS if multiple instances of it were opened *(permitted only one instance to open at a time)*
- Changes to `HOW_TO_RUN.md` to make it easier for people to be able to run this locally 
- External link handling implemented *(external links will now open in default browser)*

### CODE REVIEW

**Review by TheOtterlord — COMMENTED** [review]
~~This is the wrong way round isn't it?~~ nvm, read it wrong XD

File: baklava/src/electron.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ function createWindow() {
     mainWindow.webContents.openDevTools();
   }
   mainWindow.loadURL(
-    __prod__ ? `https://dogehouse.tv/` : "https://dogehouse.tv/"
+    __prod__ ? `https://dogehouse.tv/` : "http://localhost:3000/"
```
Comment (TheOtterlord) [inline baklava/src/electron.ts]: Here

**Comment by amitojsingh366** [comment]
How did i manage to spell electron wrong twice....

File: baklava/src/electron.ts (hunk 1 above)
Comment (amitojsingh366) [reply baklava/src/electron.ts]: no that's correct

Comment (amitojsingh366) [reply baklava/src/electron.ts]: if `__prod__` is `true` choose the first URL otherwise choose the 2nd url

**Pull request merged**